src/beamlime/services/dashboard.py

```diff
@@ -201,17 +194,49 @@ def create_monitor_plot(key: str, data: sc.DataArray) -> go.Figure:
         )
         return fig
 
+    @staticmethod
+    def create_detector_plot(key: str, data: sc.DataArray) -> go.Figure:
+        if len(data.dims) == 1:
+            return DashboardApp.create_monitor_plot(key, data)
+
+        fig = go.Figure()
+        x_dim, y_dim = data.dims
+        fig.add_heatmap(
+            z=[[]],
+            x=[],  # Will be filled with coordinate values
+            y=[],  # Will be filled with coordinate values
+            colorscale='Viridis',
+        )
+
+        def maybe_unit(dim: str) -> str:
```

Comment:
Use the `maybe_unit` in the monitor plot above as well?

Reply:
Monitors always have a time axis with a unit.